resources/install.rb

```diff
@@ -179,6 +179,16 @@
         baseurl      repo_url
         gpgkey       repo_signing_key
       end
+      execute 'disable-nginx-module' do
+        command 'dnf -y module disable nginx'
+        only_if { node['platform_version'].to_i >= 8 && platform_family?('rhel') }
+        not_if 'dnf module list nginx | grep -q "^nginx.*\[x\]"'
+      end
+      execute 'disable-nginx-module' do
+        command 'dnf -y module disable nginx'
+        only_if { platform_family?('fedora') }
+        not_if 'dnf module list nginx | grep -q "^nginx.*\[x\]"'
+      end
```

Comment:
This would be better
```suggestion
      execute 'dnf -y module disable nginx' do
        only_if do
          (node['platform_version'].to_i >= 8 && platform_family?('rhel') }) || platform_family?('fedora')
        end
        not_if 'dnf module list nginx | grep -q "^nginx.*\[x\]"'
      end
```